include/zio/zio_dev.h

```diff
@@ -141,6 +147,37 @@ struct zio_dev_chan {
 
 };
 
+/**
+ * @brief Pre-defined attribute types with the ability to extend by a driver
+ * implementation.
+ *
+ * Extended values should always be based on ZIO_ATTR_TYPES count with an
+ * additive and defined in a publicly available driver header.
+ *
+ * Example: #define MY_ATTR_TYPE (ZIO_ATTR_TYPES+1)
+ */
+enum zio_attr_type {
+	/* samples per second */
```

Comment:
Enum records generally have the comment **after** the field, with `/**<` to indicate that the comment block refers to the preceding field, so:

```
ZIO_SAMPLE_RATE,
/**< @brief samples per second */
...
```